### PR TITLE
Update gravity bridge deployment guides

### PR DESCRIPTION
**Pull request by mofhusseini**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [x] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [x] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [x] Have you rebased your work on top of the latest master? 
- [x] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: docs/gravity-bridge/gravity-bridge-mainnet-deployment-guide.md [hunk 1]
```diff
@@ -1,64 +1,89 @@
 
-  
-# Gravity Bridge Deployment Guide
 
-This guide is intended to assist the community validators with setting up crypto.org Gravity Bridge against `Cronos Testnet 3`  
+# Gravity Bridge Mainnet Deployment Guide
```
Comment (thomas-nguy) [inline docs/gravity-bridge/gravity-bridge-mainnet-deployment-guide.md:3]: This document will need to be updated later on. 

This won't work with the current mainet binary

Maybe add some disclosure that this document is for a "future" mainnet

**Review by thomas-nguy — APPROVED** [review]

File: docs/gravity-bridge/gravity-bridge-mainnet-deployment-guide.md (hunk 1 above)
Comment (mofhusseini) [reply docs/gravity-bridge/gravity-bridge-mainnet-deployment-guide.md:3]: @thomas-nguy maybe remove the mainnet guide for now? I don't think it's needed

Comment (thomas-nguy) [reply docs/gravity-bridge/gravity-bridge-mainnet-deployment-guide.md:3]: seems reasonable

**Comment by yihuang** [comment]
it closes #78

**Review by yihuang — APPROVED** [review]

**Pull request merged**